src/store/modules/site_config.js

```diff
@@ -187,6 +187,13 @@ const getters = {
     return fwo
   },
 
+  // Available automatic 3-color image options
+  auto_color_options: (state, getters) => {
+    const fwo = getters.selected_filter_wheel_config.settings?.auto_color_options
```

Comment:
fwo (from the method above) refers to filter wheel options (admittedly not a fantastic name). I think a different name here would help future readability.